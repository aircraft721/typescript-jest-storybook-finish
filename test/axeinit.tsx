import initStoryshots from '@storybook/addon-storyshots';
import { axe, toHaveNoViolations } from 'jest-axe';
import { mount } from 'enzyme';

initStoryshots({
    framework: 'react',
    test: async ({ story, context }) => {
        const component = story.render(context);
        const wrapper = mount(component);
        expect(await axe(wrapper.html())).toHaveNoViolations();
    }
});