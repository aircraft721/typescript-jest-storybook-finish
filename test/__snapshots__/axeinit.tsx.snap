// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots App app 1`] = `
<div>
  <h1>
    Hello 
    Michael
  </h1>
</div>
`;

exports[`Storyshots Black Diamond Woo hoo 1`] = `
<div>
  <p>
    Woooo BLACK DIAMOND
  </p>
  <strong>
    yep yep
  </strong>
</div>
`;

exports[`Storyshots Wakka wakka 1`] = `
<div>
  <p>
    Wakka flokka in da house
  </p>
  <strong>
    yep yep
  </strong>
  <img
    src="fifa.png"
  />
</div>
`;
